src/components/StudioCard/StudioCard.js

```diff
@@ -1,7 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
+import {
+  Card,
+  ImgBox,
+  Content,
+  ContentPrice,
+  Title,
+  StyledSlider,
+  StyledLink,
+  WishBtn,
+  OutlineHeart,
+  FillHeart,
+} from './StyleStudioCard';
 
-const StudioCard = () => {
-  return <div>StudioCard</div>;
+const StudioCard = ({ list, settings }) => {
+  const [isWishlistAdd, setIsWishlistAdd] = useState(false);
+
+  const handleWishBtn = () => {
+    setIsWishlistAdd(!isWishlistAdd);
```

Comment:
```suggestion
    setIsWishlistAdd(prev => !prev);
```
staet 업데이트 시에 함수형 업데이트로 해주시는 것이 좋습니다!